lib/ecom/types.ts

```diff
@@ -105,4 +105,6 @@ export type EcomApi = {
     logout: (returnUrl: string) => Promise<{ logoutUrl: string }>;
     isLoggedIn: () => boolean;
     getUser: () => Promise<Member | undefined>;
+    updateUser: (id: string, user: members.UpdateMember) => Promise<void>;
+    resetPassword(email: string): Promise<void>;
```

Comment:
maybe name it accordingly to what it does - `sendPasswordResetEmail`?

Reply:
renamed